bin/inx-chronicle/src/api/stardust/analytics/routes.rs

```diff
@@ -29,7 +30,8 @@ pub fn routes() -> Router {
             Router::new()
                 .route("/storage-deposit", get(storage_deposit_analytics))
                 .route("/native-tokens", get(unspent_output_analytics::<FoundryOutput>))
-                .route("/nfts", get(unspent_output_analytics::<NftOutput>)),
+                .route("/nfts", get(unspent_output_analytics::<NftOutput>))
+                .route("/richlist", get(richlist_analytics)),
```

Comment:
Nitpick: Probably something like `richest-addresses` is more appropriate? If we change it here, we should also change the other method calls.

Reply:
I dunno, it's more than that. I think richlist is appropriate.